MessageTerminal/MessageTerminal.cxx

```diff
@@ -103,11 +107,11 @@ void get_status(istringstream& iss)
             }
```

Comment:
Should we have `STATUS_FAULT`?

Reply:
Sorry forgot about this, it's coming up in the next PR